docs/source/contributors-guide/frontend.rst

```diff
@@ -0,0 +1,26 @@
+========
+Frontend
+========
+
+ViRelAy consists of 2 parts, the backend written in Python using Flask, and the frontend implemented using Angular. A production-ready version of the frontend is included in the repository, so it can be directly served by Flask. In case the frontend needs to be compiled, first the dependencies need to be installed using:
+
+.. code-block:: console
+
+    cd virelay/frontend
+    npm install
```

Comment:
Why did you remove the `$` at the beginning of the line?
The copy button automatically removes the character from the copied text, it highlights that the code is console code, and with the `$` you can highlight which part is command, and which is output.

Reply:
Okay, I did not know that the copy button removed the `$` and the argument that it makes it easier to distinguish between command and output is a good one. I hadn't considered this, so I am going to reinsert them.